File: Algoritmos Computacionales/coordenadas.py
import numpy as np
print("Ingresa la cooredenada r")
r = float(input())
print("Ingresa la cooredenada theta")
theta = float(input())
print("Ingresa la cooredenada phi")
phi = float(input())
theta = theta*np.pi/180 #cambio a radianes
phi = phi*np.pi/180
x = r*np.sin(phi)*np.cos(theta)
y = r*np.sin(phi)*np.sin(theta)
z= r*np.cos(phi)
print("Las coordenadas cartesianas son:\n(x,y,z) =", (x,y,z))

File: Algoritmos Computacionales/coordenadas.jl
#pedimos coordenadas
println("Ingresa la cooredenada r")
r = parse(Float64,readline())
println("Ingresa la cooredenada theta")
theta = parse(Float64,readline())
println("Ingresa la cooredenada phi")
phi = parse(Float64,readline())
#cambio a radianes
theta = theta*pi/180
phi = phi*pi/180
#calculamos el cambio de coordenadas
x = r*sin(phi)*cos(theta)
y = r*sin(phi)*sin(theta)
z= r*cos(phi)
#imprimimos los resultados
println("Las coordenadas cartesianas son:\n(x,y,z) =", (x,y,z))
